### PR TITLE
UIDATIMP-1463 translations

### PR DESCRIPTION
**Pull request by JohnC-80**
Translations to cover the bulk of Data Import Task Force work.

Once Localize runs with these keys in place, these translations will be copied to the orchid branch for initial rollout of work.

These translations include:
* A message that large MARC files will be split to separate parts.
* An error message specifically for errors while uploading files.
* Composite job status information: Parts processed, remaining, completed, completed with errors, failed
* Part number / total number for composite jobs in the Job Log entry rows.

### CODE REVIEW

File: translations/ui-data-import/en.json [hunk 1]
```diff
@@ -23,6 +23,11 @@
   "beganRunning": "Began {time}",
   "endedRunning": "Ended {time}",
   "progressRunning": "In progress",
+  "jobProgress.partsRemaining": "Job parts remaining: {current} of {total}",
+  "jobProgress.partsProcessed": "Job parts processed: {current} of {total}",
+  "jobProgress.partsCompleted": "Completed: {amount}",
+  "jobProgress.partsCompletedWithErrors": "Completed with Errors: {amount}",
```
Comment (ncovercash) [inline translations/ui-data-import/en.json:29]: Nitpick: shouldn't **E**rrors be lowercase?

File: translations/ui-data-import/en.json [hunk 2]
```diff
@@ -23,6 +23,11 @@
   "beganRunning": "Began {time}",
   "endedRunning": "Ended {time}",
   "progressRunning": "In progress",
+  "jobProgress.partsRemaining": "Job parts remaining: {current} of {total}",
+  "jobProgress.partsProcessed": "Job parts processed: {current} of {total}",
+  "jobProgress.partsCompleted": "Completed: {amount}",
```
Comment (ncovercash) [inline translations/ui-data-import/en.json:28]: Should `{amount}` be `{amount} parts`?  If I just see `Completed: 25` as a user it might be a bit confused

**Review by ncovercash — CHANGES_REQUESTED** [review]

File: translations/ui-data-import/en.json (hunk 2 above)
Comment (JohnC-80) [reply translations/ui-data-import/en.json:28]: for context, these are a 'subtotals' section below the parts/totals count...
image

Comment (ncovercash) [reply translations/ui-data-import/en.json:28]: ahh ok, sounds good!

**Review by ncovercash — APPROVED** [review]
LGTM

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**